main.tf

```diff
@@ -0,0 +1,88 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 3.8"
```

Comment:
(Not Important) Probably worth bumping up this number (`4.19.0` at the time of writing).